src/less-style/tree-content.less

```diff
@@ -115,7 +111,6 @@
 
 .fd-head-external-sources {
   background-color: @cc-neutral-low;
-  z-index: 0;
```

Comment:
This was added as a bugfix for the language dropdown appearing below the header (https://github.com/dimagi/Vellum/commit/c2f7777404da8edb46b5940a67d6446908210cfc), isn't necessary now that `.fd-head` doesn't have a z-index.